src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -162,7 +162,7 @@
   </Node>
 
   <Node Name="BoundTypeExpression" Base="BoundExpression">
-    <Field Name="AliasOpt" Type="AliasSymbol" Null="allow"/>
```

Comment:
>" Null="allow" [](start = 44, length = 14)

I find the removal of some `Null="allow"` but not other `Null` properties a bit disturbing. Should we keep it instead of the `?` and have the generator produce the `?` instead?

Reply:
I initially tried it that way and it got really hairy because the Type that was written wasn't necessarily what was emitted.

I think the current approach is lossless -- everything without a `Null` specification is annotated as the Type is written. We should rely on nullability warnings to refine the specification as we go forward.